contributors/design-proposals/api-proposal-design-template.md

```diff
@@ -0,0 +1,84 @@
+# Description of your Api
+
+## Owners
+
+Sig owner: list of sigs that will own the design, implementation, and support
+Individual owners: list of individual GitHub users that will own the design, implementation, and support
```

Comment:
I think we need similar info as suggested for the features template:
https://github.com/kubernetes/features/pull/164/files

In particular, we need assigned+agreed reviewers and approvers.